### ISSUE TITLE
ad_dif_conductance must be manually updated when running AdvectionDiffusion.run()

### ISSUE THREAD

**Issue by ma-sadeghi** [PMEAL/OpenPNM#1372]
The problem is that `ad_dif_conductance` depends on `phase["pore.pressure"]`, which is only available after running `StokesFlow.run()`. So, in a normal OpenPNM simulation workflow, since `physics` object is obviously instantiated before running `StokesFlow`, the value of `ad_dif_conductance` is outdated since it has been calculated using obsolete `phase["pore.pressure"]` values.

There are many workarounds, here are two alternatives that I thought about:
1. Set `throat.ad_dif_conductance` as an `iterative_prop`, so it gets updated prior to calling the solver. This has the unintended effect of needlessly re-calculating `ad_dif_conductance` in iterative workflows (ex. nonlinear source term, etc.).
2. Overload `run` method for `AdvectionDiffusion` like this:
```
def run():
    self.physics.regenerate_models(propnames="throat.ad_dif_conductance")
    self.super().run()
```

**Comment by ma-sadeghi** [comment]
@jgostick @magnaou What do you think?

**Comment by ma-sadeghi** [comment]
This is now addressed in PR #1391.
